react/features/overlay/components/OverlayContainer.js

```diff
@@ -14,6 +14,42 @@ import UserMediaPermissionsFilmstripOnlyOverlay
     from './UserMediaPermissionsFilmstripOnlyOverlay';
 import UserMediaPermissionsOverlay from './UserMediaPermissionsOverlay';
 
+/**
+ * Reference to the lazily loaded list of overlays.
+ */
+let _overlays;
```

Comment:
I ended up doing this because the CallOverlay was undefined at this point, but if accessed later it's not. What gives?

Reply:
Others have diagnosed similar problems as being caused by circular dependencies. (CircularDependencyPlugin shows that master has 157 circular dependencies.)

Reply:
Thanks Lenny! Alas I don't see anything obvious that would cause this :-S

Reply:
The undefined import values are indeed the result of circular references.

But they shouldn't be a real problem for us because they are undefined only at the global module level. And we discourage execution there for the sake of the mobile app. Technically, `interfaceConfig` is a value that is not immutable because mobile would download it from the deployment and the value may change between deployments. So `_overlays` shouldn't (one day) be global. I know that practically today the source code will perform correctly (given that it's related to `interfaceConfig` and `filmstripOnly` which are Web-only features at this time) but I discourage it for the sake of maintainability and (future) extensibility.